### PR TITLE
Zanzana: Run OpenFGA HTTP server in standalone mode

### PR DESCRIPTION
**Pull request by alexanderzobnin**
**What is this feature?**

Run OpenFGA HTTP server when zanzana is running as grafana module (standalone).

**Why do we need this feature?**

It allows to connect and use `fga` CLI tools.

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/identity-access-team/issues/760

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: pkg/setting/settings_zanzana.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ type ZanzanaSettings struct {
 	Addr string
 	// Mode can either be embedded or client
 	Mode ZanzanaMode
+	// ListenHTTP enables OpenFGA http server which allows to use fga cli
+	ListenHTTP bool
```
Comment (kalleep) [inline pkg/setting/settings_zanzana.go:20]: I wonder if we should change this to a full http address `HttpAddr` so you can specify on what address to listen on

**Review by kalleep — COMMENTED** [review]
nice, it is for sure a useful feature to have. Would it be possible to the http api when running embedded version, we can either try to integrate it with existing server or start another one

**Review by Jguer — APPROVED** [review]

File: pkg/services/authz/zanzana.go [hunk 2]
```diff
@@ -136,6 +136,16 @@ func (z *Zanzana) start(ctx context.Context) error {
 }
 
 func (z *Zanzana) running(ctx context.Context) error {
+	go func() {
+		if z.cfg.Zanzana.ListenHTTP {
```
Comment (Jguer) [inline pkg/services/authz/zanzana.go:140]: maybe we should add a cfg.Environment == settings.Dev check here so it's not accidentally started in prod envs

File: pkg/setting/settings_zanzana.go (hunk 1 above)
Comment (alexanderzobnin) [reply pkg/setting/settings_zanzana.go:20]: Yes, I need to make it configurable

**Comment by alexanderzobnin** [comment]
@kalleep problem with embedded version is connection to GRPC. We use channels to communicate, not sure how to pass it to the `RegisterOpenFGAServiceHandler()` which wants `*grpc.ClientConn`. I'll take a look.

**Pull request merged**